### PR TITLE
Added demo for colorpicker

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-colorpicker-demo](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-colorpicker-demo)

**Review by Blackbaud-DeniseCisneros — DISMISSED** [review]

File: src/app/docs/colorpicker/colorpicker-docs.component.html [hunk 1]
```diff
@@ -8,14 +8,38 @@
...
+    [packageDependencies]="{
+      '@skyux/colorpicker': '*'
+    }"
+  >
+    <sky-docs-code-example
+      heading="Reactive colorpicker"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/colorpicker/reactive"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Template-driven colorpicker"
```
Comment (Blackbaud-SteveBrush) [inline src/app/docs/colorpicker/colorpicker-docs.component.html:37]: This was a discussion I had with Todd/Adam several months ago, but we've decided to remove the template-driven forms entirely and only demo the reactive forms.

Comment (Blackbaud-AlexKingman) [reply src/app/docs/colorpicker/colorpicker-docs.component.html]: Noted. I've removed this.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-colorpicker-demo](https://developer.blackbaud.com/skyux-colorpicker?_sp.spa=pr-colorpicker-demo)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**